File: hw6/hw6.py
def make_playlist(songs, artists):
    playlist = {}
    for i in range(len(songs)):
        playlist[songs[i]] = artists[i]
    
    return playlist

print(make_playlist(["Tầng Thượng 102", "Đốt", "Cho Tôi Đi Theo"], ["CHH", "Ngọt", "Ngọt"]))
        

File: hw6/HW6.java
package hw6;

import java.util.Map;
import java.util.HashMap;

public final class HW6 {
    private HW6() {
        //
    }

    public static Map<String, String> makePlaylist(String[] songs, String[] artists) {
        Map<String, String> playlist = new HashMap<>();
        for (int i = 0; i < songs.length; i++) {
            playlist.put(songs[i], artists[i]);
        }

        return playlist;
    }
}
